Houdini/config/packages/dev/routing.yaml

```diff
@@ -1,3 +0,0 @@
-framework:
-    router:
-        strict_requirements: true
```

Comment:
Threw me for a loop at first, but yeah, makes sense... assuming they changed the default at some point from `NULL` to [`TRUE`](https://symfony.com/doc/current/reference/configuration/framework.html#strict-requirements)... so the default of `TRUE` here and on `test`, and then `NULL` being spec'd explicitly on the `prod` bit. 👍

Reply:
Yeah sorry, I just noticed that Houdini was different because it was migrated to Flex before the rest. Tried to get them consistent.

Reply:
No worries.

Reply:
Looking back in the history of this parameter, it seems it was always a default of `true` so this might have just been a mistake of theirs from before.